### DISCUSSION TITLE
@percy/cli stable version

### DISCUSSION THREAD

**Discussion by pustovalov** [percy/percy-storybook#392] [Support] [answered]
Hey,

Since `@percy/storybook@^4` uses `@percy/cli` https://github.com/percy/percy-storybook#upgrading

WDYT about to create stable version of `@percy/cli` ?

**Comment by wwilsman** [comment, marked as answer]
Hey @pustovalov!

See this issue for the remaining SDKs that need to be updated before we un-beta the CLI (just one more to go): https://github.com/percy/cli/issues/57

Besides that, there are a small handful of other features and bug fixes we have planned before we un-beta the CLI. It's pretty stable as-is, but will remain as a "beta" until those tasks are complete.
